mantidproject/mantid
# Move frequency domain analysis to workbench

Merge after #25573

**Description of work.**

This moves the Frequency Domain Analysis Tab to the workbench.

**To test:**

In the workbench open the FDA GUI. It is in the Muon subsection of the interface list. Check that a few standard workflows work. If you need an explanation of how the GUI is supposed to work feel free to message me.

Fixes #25576

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
